examples/additional_examples/mixed_energy_force/methane.py

```diff
@@ -0,0 +1,108 @@
+import gaptrain as gt
+import numpy as np
+import matplotlib.pyplot as plt
+from autode.wrappers.keywords import GradientKeywords
+gt.GTConfig.n_cores = 10
+# plt.style.use('paper')
+
+
+def generate_dftb_trajectory():
+    """Generate some configurations to use as reference"""
+
+    traj = gt.md.run_dftbmd(methane.configuration(),
+                            temp=500,
+                            dt=0.5,
+                            interval=10,
+                            ps=1)
+
+    traj.save(filename='methane_dftb_md.xyz')
+    return None
+
+
+def generate_energies_forces(trajectory):
+    """Calculate energies and forces for the frames at different levels of theory"""
+
+    frames = trajectory.copy()
+    frames.parallel_dftb()
+    frames.save(filename='dftb_frames.xyz')
+    np.savetxt('dftb_frames_F.txt', frames.force_components())
```

Comment:
You have _F in this savetxt but not in the sequential files

Reply:
whoops - thanks :+1: